docker-context/Dockerfile

```diff
@@ -8,7 +8,7 @@ COPY --from=node:17.9.1 /usr/local/lib/node_modules /usr/local/lib/node_modules
 RUN ln -s ../lib/node_modules/npm/bin/npm-cli.js /usr/local/bin/npm
 RUN ln -s ../lib/node_modules/npm/bin/npx-cli.js /usr/local/bin/npx
 ARG COMPOSER_AUTH
-RUN wp --allow-root package install nlemoine/wp-cli-fixtures
+RUN wp --allow-root package install hellonico/wp-cli-fixtures
```

Comment:
I changed this to resolve the [following issue](https://github.com/orgnc/wordpress-plugin/actions/runs/4814936863/jobs/8573141395#step:7:1724):
```
#19 [builder 10/11] RUN wp --allow-root package install nlemoine/wp-cli-fixtures
#0 1.948 Installing package hellonico/wp-cli-fixtures (dev-main)
#0 1.948 Updating /root/.wp-cli/packages/composer.json to require the package...
#0 1.949 Warning: Package name mismatch...Updating from git name 'nlemoine/wp-cli-fixtures' to composer.json name 'hellonico/wp-cli-fixtures'.
#0 1.957 Registering https://github.com/nlemoine/wp-cli-fixtures.git as a VCS repository...
#0 1.984 Using Composer to install the package...
#0 1.984 ---
#0 1.988 Loading composer repositories with package information
#0 2.617 ---
#0 2.617 Warning: Could not authenticate against github.com
#0 2.618 Error: Package installation failed.
#0 2.619 Reverted composer.json.
#19 ERROR: executor failed running [/bin/sh -c wp --allow-root package install nlemoine/wp-cli-fixtures]: exit code: 1
```